### DISCUSSION TITLE
`Deno.env.*` in Island?

### DISCUSSION THREAD

**Discussion by mamunonweb** [freshframework/fresh#904] [Q&A] [answered]
### What is the proper way to expose environment variables to the client/Island?

Say, I have a client library (ex: Supabase), that has some public data to share with, that I have stored inside `.env`.

```env
// .env
PUBLIC_KEY=key
```
Now, I can use those data on the server side (inside Routes — it's not a problem).

But, how can I use those data on the client side (inside Islands)?

**Comment by Sleepful** [comment, marked as answer]
Here you go :)

https://github.com/denoland/fresh/issues/945#issuecomment-1354967538
